debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PGTableSchemaBuilder.java

```diff
@@ -0,0 +1,507 @@
+/*
+ * Copyright Debezium Authors.
```

Comment:
Why is this class required for this support ?

Reply:
This class contains the schema building methods like `create`, `createKeyGenerator`, `createValueGenerator`, etc which help in forming record structure.